addons/base_import/models/base_import.py

```diff
@@ -97,10 +97,26 @@ def get_import_templates(self):
         """
         Get the import templates label and path.
 
-        :return: a list(dict) containing label and template path
-                 like ``[{'label': 'foo', 'template': 'path'}]``
+        :return: a dict containing
+                qweb_template: name of the qweb template to display instead of the default content
+                files: list(dict) containing label and template path
+                 like ``{'qweb_template': 'module.template_id', files: [{'label': 'foo', 'template': 'path'}]}``
         """
-        return []
+        return {
+            'qweb_template': None,
+            'files': []
+        }
+
+    @api.model
+    def _get_import_sheet(self, sheet_names):
```

Comment:
Name is not great, especially since this is a fallback (if the client does not specify the sheet they want), it's more of a default.

The interface also seems rather faillible and less than convenient, as *every override* has to (remember to) search the entire sequence of sheet names in order to ensure the name they're looking for is in there.

Maybe a better interface would be for modules to specify the keys they prefer, then base_model_import can check if one of them's in there, and fallback on the first sheet if not? That way e.g. account just has to 

    return ['Contacts']

or something along those lines.

That's especially important because I really don't think people will usually rename the first sheet of their clients list "Contacts".